dagmc_stats/dagmc_stats.py

```diff
@@ -275,7 +275,7 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
     return t_a_r
 
 
-def get_area_triangle(my_core, meshset, geom_dim):
+def get_area_triangle(my_core, meshset, geom_dim, tris=[]):
```

Comment:
A test needs to be added to reflect this change. Currently only no `tris` argument is tested, but one needs to be added that tests if a subset of `tris` is added.

Reply:
Also on that test, we should be checking the values of `obs` vs `exp`, not just the length of the data returned.